infra/modules/helm/charts/templates/service-configs.yaml

```diff
@@ -17,6 +17,10 @@ kind: ConfigMap
 metadata:
   name: service-configs
 data:
+  PROJECT_ID: "{{ .Values.project_id }}"
+  SPANNER_ID: "{{ .Values.spanner_id }}"
+  DATABASE: "{{ .Values.database }}"
+  SPANNER_EMULATOR_ENABLED: "false"
```

Comment:
Don't think we'll need this, since we'd only use the emulator locally and can configure it on the dev's machine.

Reply:
Removed this in https://github.com/GoogleCloudPlatform/terraform-example-java-dynamic-point-of-sale/pull/8/commits/872dfcf1ddbe9e4214915941dd68303e6ed4bb83